CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_InfiltrationActions.uc

```diff
@@ -32,12 +32,12 @@ static function X2CovertInfiltrationTemplate CreateInfiltrationTemplate(name Cov
 
 	if (bCreateSlots)
 	{
+		Template.Slots.AddItem(CreateDefaultOptionalSlot('InfiltrationStaffSlot'));
```

Comment:
?

Reply:
Both the optional slots being on one side of the loadout screen has annoyed me forever and with Beta 1 coming up I'm fixing it.

Reply:
Then it's not gonna work due to how weird the slots are setup. Check `X2CovertInfiltrationTemplate`